test/parallel/test-fs-stat.js

```diff
@@ -72,14 +72,10 @@ fs.open('.', 'r', undefined, common.mustCall(function(err, fd) {
 // fstatSync
 fs.open('.', 'r', undefined, common.mustCall(function(err, fd) {
   let stats;
-  try {
-    stats = fs.fstatSync(fd);
-  } catch (err) {
-    assert.fail(err);
-  }
   if (stats) {
```

Comment:
With lines 75-79 removed, `stats` can't possibly be anything other than `undefined` at this point. I think you've nullified the test. The whole point of this test is to check the functionality of `fstatSync()` but that function now isn't being called.